app/engines/domain_scope_options_builder.rb

```diff
@@ -3,12 +3,19 @@ def self.build(site:, site_limits: nil, collection: nil)
     { included_domains: collection ? included_domains(collection) : site.site_domains.pluck(:domain),
       excluded_domains: site.excluded_domains.pluck(:domain),
       scope_ids: site.scope_ids_as_array,
-      site_limits: site_limits }
+      site_limits: strip_sitelimit_protocol(site_limits) }
   end
 
   def self.included_domains(collection)
     collection.url_prefixes.map do |url_prefix|
       UrlParser.strip_http_protocols(url_prefix.prefix)
     end.presence
   end
+
+  def self.strip_sitelimit_protocol(site_limits)
+    return if site_limits.blank?
+
+    site_limits.split(/\+/).
```

Comment:
Let's split on a simple string (`'+'`), in the spirit of only using Regexes when we need to (it also mirrors the re-joining with `'+'` at the end). I would also break this method up a bit differently for readability:
```ruby
    site_limits.split('+').map do |site_limit|
      UrlParser.strip_http_protocols(site_limit)
    end.join('+')
```